src/cls/IPM/Main.cls

```diff
@@ -2951,22 +2953,36 @@ ClassMethod DisplayModules(ByRef pList, pNumbered As %Boolean = 0, pWithNamespac
     $ListBuild("Repository", $$$Blue)
   )
 
+  Kill serverVersions
+  If $Data(pModifiers("showupstream"), repoName) && ##class(%IPM.Repo.Definition).ServerDefinitionKeyExists(repoName) {
+    Set query = "SELECT Name,Version FROM %IPM_Utils.Module_GetModuleList(?,?) "
+    Set rs = ##class(%SQL.Statement).%ExecDirect(, query, repoName, 0)
+    While rs.%Next() {
+		Set serverVersions(rs.%Get("Name")) = rs.%Get("Version")
+	}
+  }
   Set width = $Get(pList("width")) + 1
   Set tIndent = pIndent
   Set tIndent = $Select(pIndent > 0: pIndent, 1: width)
   For i=1:1:pList {
     Set info = pList(i)
     Set developerMode = 0
     Set root = ""
-    Set $ListBuild(name, version, externalName, developerMode, root) = info
+    Set $ListBuild(name, version, externalName, developerMode, root, lastUpdated) = info
     Write:i>1 !,?pIndent
     Write $$$FormattedLinePadRight($$$Green, name, width), $$$FormattedLine($$$Blue, version)
+    If $Data(pModifiers("showupstream")) && ($Data(serverVersions(name), version) # 2) {
+      Write " ", $$$FormattedLine($$$Blue, "(ServerSide: "_version_")")
```

Comment:
I'd consider saying "latest available" here rather than "ServerSide" which is confusing.

Reply:
Opinion: should only show this if there's a newer version available upstream.

Reply:
I was think package developers may be interested in knowing if the package version published on the registry is older than their current dev version.